### ISSUE TITLE
Customizer: Widgets creating empty "block settings" sections

### ISSUE THREAD

**Issue by Nic-Sevic** [Automattic/wp-calypso#56507]
### Quick summary

There are broken block settings sections in the customizer that are tied to unknown widgets

### Steps to reproduce

1. Open the customizer on a site with a theme with widget areas
2. add a few widgets
3. See that some (unclear which) create a block settings area
4. open a section and you can see where the widget supposedly is but not which widget or any options for settings

### What you expected to happen

these either shouldn't be there or should id the widget and contain settings

### What actually happened

there are blank sections but not clear why

![2021-09-23 at 09 18](https://user-images.githubusercontent.com/47717887/134524271-0df8a98c-8ac2-4a33-997f-f2a00155ab71.jpg)
![2021-09-23 at 09 18](https://user-images.githubusercontent.com/47717887/134524308-dc5f3875-b858-4ddf-82fd-2f0f2e030824.jpg)

### Context

first saw on user site but not user specific

### Other notes

happening on both AT and simple

### Severity

All

**Comment by mrfoxtalbot** [comment]
I came in to report exactly this. I have taken the liberty to move this issue to the "triaged" column to have it addressed as soon as possible.

**Comment by Robertght** [comment]
Thank you @Nic-Sevic & @mrfoxtalbot ! This is already being looked into here: https://github.com/WordPress/gutenberg/pull/35055

cc @mreishus

**Comment by kuropixel** [comment]
28359837-hc
A user came in with this issue. Atomic, using Astra theme.
![block settings](https://user-images.githubusercontent.com/83020905/134831262-939bc340-0bcd-44ba-95f8-31ed0ccbc572.png)
No work arounds?

**Comment by philnick206** [comment]
Noticed this in the following as well #4332016-zen

**Comment by benchilcote** [comment]
Reporting another case of this #4333521-zen

(not to mention noticing it on my test websites)

**Comment by thekingsprojects** [comment]
Another case here: 3147804-hc

Additionally, this site seems to be having trouble loading the actual "Widget" option; I had to wrestle with trying to access Customizer/Widgets through WP-admin to eventually get the proper widget area settings to load, and it wasn't always consistent.

**Comment by sixhours** [comment]
@mreishus Since we're waiting on Gutenberg here, what do you think about us adding a CSS `display: none` hack or similar on the Jetpack/WP.com side?

**Comment by mreishus** [comment]
We have a fix merged into gutenberg now: https://github.com/WordPress/gutenberg/commit/b9ad37d4007dba76279fadd9001d3e41b786daad
Looking into non-disruptive workarounds.

**Comment by mreishus** [comment]
D67999-code : Needs review

**Comment by mreishus** [comment]
D67999 has been deployed; let me know if this is still an issue.

**Comment by ccwalburn** [comment]
Hi @mreishus 
I got a report here: 32124722-hc (follow up 4365464-zen)
Deactivating Gutenberg removed the Block Settings from the Customizer.
They appeared again when I reactivated Gutenberg so I have left it deactivated for now.

**Comment by kingsleyinfo** [comment]
Another report here: 32142904-hc
Tried disabling Gutenberg but that removed all the Block Settings error but they still cannot do anything with the customizer.

**Comment by sixhours** [comment]
Hmm, I wonder if the CSS we added doesn't account for how all themes name their widget areas...that could account for this bug still occurring. Let's wait until Gutenberg is merged to WP.com (tomorrow, I believe) to confirm it's still an issue.

**Comment by sixhours** [comment]
The latest Gberg is live on WP.com and Atomic sites, so in theory this should be fixed @kingsleyinfo @ccwalburn let us know if it's not. Thank you!

**Comment by ccwalburn** [comment]
Thanks @sixhours ! I tested my user's site and everything looks good :)
